Libraries/LibRegex/RegexOptimizer.cpp

```diff
@@ -1176,8 +1176,8 @@ void Optimizer::append_alternation(ByteCode& target, Span<ByteCode> alternatives
             patch_locations.append({ node_ip, target_ip });
         };
 
-        Queue<Tree*> nodes_to_visit;
-        nodes_to_visit.enqueue(&trie);
+        Vector<Tree*> nodes_to_visit;
```

Comment:
You should be able to replace `#include <AK/Queue.h>` with `#include <AK/Vector.h>` now.